data/types/pumpstatus/reservoirremaining.go

```diff
@@ -0,0 +1,42 @@
+package pumpstatus
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinReservoirAmount = 0
+	MaxReservoirAmount = 1000
+)
+
+type ReservoirRemaining struct {
+	Unit   *string  `json:"units,omitempty" bson:"units,omitempty"`
+	Amount *float64 `json:"amount,omitempty" bson:"amount,omitempty"`
```

Comment:
Doesn't match spec. Spec has `startDate` (which seems redundant with `Base`s `time` field?) and no `Unit`.

Reply:
ok - just made this a number on the pumpStatus object.